lioncore/metametamodel/lioncore.json

```diff
@@ -0,0 +1,2196 @@
+{
+  "serializationFormatVersion": "1",
+  "languages": [
+    {
+      "key": "LIonCore-M3",
+      "version": "1"
+    }
+  ],
+  "nodes": [
+    {
+      "id": "-id-LIonCore-M3",
```

Comment:
All id values start with a minus, is this really the intention? Or is something missing?

Reply:
I deliberately used different _ids_ and _keys_, so it's clear everywhere which one we refer to.
In principle the _id_ should be arbitrary (as long as they resolve). As I wrote this file by hand, I used kind of meaningful ids.